static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -39,6 +39,18 @@ type Props = {
 
 function SetupDocs({organization, projects, search}: Props) {
   const api = useApi();
+  const [clientState, setClientState] = useState<ClientState | null>(null);
+  const selectedProjectsSet = clientState
```

Comment:
@Neo-Zhixing Maybe optional chaining would be less verbose here

Reply:
Ok but then you're potentially calling `new Set(undefined)` which is very weird.

Reply:
@Neo-Zhixing you can add a `|| []` to prevent `new Set(undefined)`